neptune/experiment.py

```diff
@@ -358,16 +383,16 @@ def __ne__(self, o):
     def _simple_dict_to_dataframe(d):
         return pd.DataFrame.from_dict(map_values(lambda x: [x], d))
 
-    def _send_channel_value(self, name, channel_type, x, y):
-        channel = self._get_channel(name, channel_type)
+    def _send_channel_value(self, channel_name, channel_type, x, y):
+        channel = self._get_channel(channel_name, channel_type)
 
         if x is None:
             if channel.x is None:
                 channel.x = 0
             x = channel.x + 1
         elif x <= channel.x:
             raise ValueError("ValueError: X-coordinates must be strictly increasing. "
-                             "Invalid Point({}, {}) for channel \"{}\"".format(x, y, name))
+                             "Invalid Point({}, {:100.100}) for channel \"{}\"".format(x, y, channel_name))
```

Comment:
Co robi :100.100?

Reply:
Limits `y` formatted value to at most 100 characters